src/Computer.php

```diff
@@ -164,36 +164,37 @@ public function post_updateItem($history = 1)
         $input = Toolbox::addslashes_deep($this->fields);
         for ($i = 0; $i < $update_count; $i++) {
            // Update contact of attached items
-            if ($this->updates[$i] == 'contact_num' && $CFG_GLPI['is_contact_autoupdate']) {
+            if ($this->updates[$i] == 'contact_num' && Entity::getUsedConfig('is_contact_autoupdate', $this->getField('entities_id'))) {
```

Comment:
`getField()` was meant to be used only on views. Cannot be deprecated due to its many uses, but it should not be used anymore, except on a few cases.

```suggestion
            if ($this->updates[$i] == 'contact_num' && Entity::getUsedConfig('is_contact_autoupdate', $this->getEntityID())) {
```